mucgpt-frontend/src/components/ChatUsageIndicator/ChatUsageIndicator.module.css

```
@@ -0,0 +1,148 @@
+ */
+.ringButton {
+    min-width: 0;
+    width: 40px;
+    height: 40px;
+    padding: 0;
+    overflow: visible;
+}
+
+.ring {
+    --usage-percent: 0;
+    position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)